python/ray/serve/api.py

```diff
@@ -232,6 +238,23 @@ def create_backend(
 
         if config is None:
             config = {}
+        if ray_actor_options is None:
+            ray_actor_options = {}
+        if env is None:
+            # If conda is activated, default to conda env of the driver.
+            if os.environ.get("CONDA_PREFIX"):
+                ray_actor_options.update(override_environment_variables={
+                    "PYTHONHOME": os.environ.get("CONDA_PREFIX")
+                })
```

Comment:
Why do we need to change anything in this case? Shouldn't it just work out of the box (i.e., how it currently does)?

Reply:
Ah, by "driver" in the comment, I mean the process that called create_backend--I'll edit to make it more clear.  If a user is working in a tf1 env and runs a script that starts a tensorflow backend (without providing an `env` parameter), then the user would expect that backend to run in the tf1 env, regardless of which env the controller driver happens to be running in.  That's what this change does.  

The alternative would be to let it work the way it currently does, where the backend would just run in whatever env the controller driver happens to be running in.